core/sqf/sqenvcom.sh

```diff
@@ -47,6 +47,15 @@ export PRODUCT_COPYRIGHT_HEADER="2015-2017 Apache Software Foundation"
 ##############################################################
 export TRAFODION_ENABLE_AUTHENTICATION=${TRAFODION_ENABLE_AUTHENTICATION:-NO}
 
+# Set the Trafodion Configuration store type
+if [[ -n "$CLUSTERNAME" ]]; then
+  # This is a cluster environment, not a workstation
+  #export TRAF_CONFIG_DBSTORE=MySQL
+  export TRAF_CONFIG_DBSTORE=Sqlite
+else
+  export TRAF_CONFIG_DBSTORE=Sqlite
```

Comment:
Looks like it is aways Sqlite. Why bother with the "if"? Earlier I saw mention in the code that Sqlite is deprecated. Is that still the case?

Reply:
This is an initial step to prepare the code base before adding the MySQL storage method.